CHANGELOG.rst

```diff
@@ -1,3 +1,9 @@
+2.0.0 (2015-XX-XX)
```

Comment:
I usually do that on the master branch after it is merged just in case anything else gets included in the new version (not likely here).